fusedrug/data/protein/structure/protein_complex.py

```diff
@@ -22,18 +22,26 @@ def __init__(self, verbose: bool = True) -> None:
         self.chains_data = {}  # maps from chain description (e.g. ('7vux', 'A')) to
         self.flattened_data = {}
 
+        #
+        self.per_chain_most_frequent_residue_part = {}
```

Comment:
The key is chain_id?

Reply:
a key is a tuple in the format (pdb_id, chain_id)

added a comment above it with description